src/components/App.js

```diff
@@ -23,13 +22,6 @@ const HTML5toTouch = {
 
 export const App = DragDropContext(MultiBackend(HTML5toTouch))(() =>
   <Provider store={store}>
-    <div onTouchMove={
-      () => globals.touching = true // eslint-disable-line no-return-assign
-    } onTouchEnd={() => {
-      globals.touching = false // eslint-disable-line no-return-assign
-      globals.touched = true // eslint-disable-line no-return-assign
-    }}>
-      <AppComponent/>
-    </div>
```

Comment:
If you remove this from the app and only add it in the tests, then it will not work in the build.

Reply:
FWIW, I think these global touch variables can be safely ignored by the tests.

Reply:
I deleted it by mistake, it's back now